src/com/serotonin/mango/vo/report/ReportChartCreator.java

```diff
@@ -122,6 +121,8 @@ public void createContent(ReportInstance reportInstance, ReportDao reportDao, St
         model.put("IMAGE", DataTypes.IMAGE);
         model.put("toYear", DateUtils.getCurrentYearInt());
 
+        pointStatistics.sort(Comparator.comparing(PointStatistics::getName));
```

Comment:
This is the wrong place for it. Look for the place where this structure is created.